app/controllers/reports_controller.rb

```diff
@@ -0,0 +1,42 @@
+class ReportsController < ApplicationController
+  def index
+    if current_instructor
+      @reports = Report.all
+    else
+      @reports = Report.where(student_id: current_student.id)
+    end
+  end
+
+  def new
+    authenticate_instructor!
```

Comment:
I think thats good, the only reason they can't currently is because I couldn't figure out what to do with the inputs that are generated by me. The 'satisfactory', 'unsatisfactory', etc.